tests/integration/update.bats

```diff
@@ -86,6 +90,16 @@ function check_cgroup_value() {
     [ "$status" -eq 0 ]
     check_cgroup_value $CGROUP_CPU "cpu.cfs_quota_us" 600000
 
+    # update cpu-rt-period
+    runc update test_update --cpu-rt-period 900000
+    [ "$status" -eq 0 ]
+    check_cgroup_value $CGROUP_CPU "cpu.rt_period_us" 900000
+
+    # update cpu-quota
+    runc update test_update --cpu-rt-runtime 600000
+    [ "$status" -eq 0 ]
+    check_cgroup_value $CGROUP_CPU "cpu.rt_runtime_us" 600000
```

Comment:
Yeah, we can't add this to the test because we don't have `rt` enabled on our janky server kernel. One of the things that we're planning on doing is splitting this test so we can have conditional cgroup testing (using `requires`), but that'll happen once someone has some spare time :wink:. In the meanwhile, just remove this from the test.